src/main/java/roomescape/reservation/ReservationService.java

```diff
@@ -28,12 +62,50 @@ private ReservationRequest replaceNameIfEmpty(ReservationRequest request, LoginM
     }
 
     public void deleteById(Long id) {
-        reservationDao.deleteById(id);
+        reservationRepository.deleteById(id);
     }
 
     public List<ReservationResponse> findAll() {
-        return reservationDao.findAll().stream()
-                .map(it -> new ReservationResponse(it.getId(), it.getName(), it.getTheme().getName(), it.getDate(), it.getTime().getValue()))
+        return reservationRepository.findAll().stream()
+                .map(reservation -> new ReservationResponse(
+                        reservation.getId(),
+                        reservation.getName(),
+                        reservation.getTheme().getName(),
+                        reservation.getDate(),
+                        reservation.getTime().getTime()
+                ))
                 .toList();
     }
+
+    public List<MyReservationResponse> findMyReservation(LoginMember loginMembmer) {
+        List<Reservation> reservations = reservationRepository.findByMemberId(loginMembmer.getId());
+        List<WaitingWithRank> waitings = waitingRepository.findWaitingsWithRankByMemberId(loginMembmer.getId());
+
+        List<MyReservationResponse> myReservationResponses = new ArrayList<>();
+        reservations.forEach(
+                reservation -> myReservationResponses.add(
+                        new MyReservationResponse(
+                                reservation.getId(),
+                                reservation.getTheme().getName(),
+                                reservation.getDate(),
+                                reservation.getTime().getTime(),
+                                "예약"
+                        )
+                )
+        );
```

Comment:
```suggestion
        List<MyReservationResponse> myReservationResponses = reservations.stream()
            .map(reservation -> new MyReservationResponse(
                reservation.getId(),
                reservation.getTheme().getName(),
                reservation.getDate(),
                reservation.getTime().getTime(),
                "예약"
            ))
            .collect(Collectors.toList());
```

이렇게 스트림을 사용할 수도 있을 것 같네요! (불변 컬렉션을 만든다는 이점이 있음)